mods/cnc/maps/gdi06/gdi06.lua

```diff
@@ -0,0 +1,159 @@
+CommandoReinforcements = { "rmbo" }
+IslandSamSites = { SAM01, SAM02 }
+NodBase = { PowerPlant1, PowerPlant2, PowerPlant3, PowerPlant4, PowerPlant5, Refinery, HandOfNod, Silo1, Silo2, Silo3, Silo4, ConYard, CommCenter}
+CivFleeRoute = { CivFleeRoute1.Location, CivFleeRoute2.Location, CivFleeRoute3.Location }
+
+FlameSquad = { FlameGuy1, FlameGuy2, FlameGuy3 }
+FlameSquadRoute = { waypoint4.Location, waypoint12.Location, waypoint4.Location, waypoint6.Location }
+
+HuntTriggerActivator = { SAM03, SAM04, SAM05, SAM06, LightTank1, LightTank2, LightTank3, Buggy1, Buggy2, Turret1, Turret2 }
+
+AttackCellTriggerActivator = { CPos.New(57,26), CPos.New(56,26), CPos.New(57,25), CPos.New(56,25), CPos.New(57,24), CPos.New(56,24), CPos.New(57,23), CPos.New(56,23), CPos.New(57,22), CPos.New(56,22), CPos.New(57,21), CPos.New(56,21) }
+AttackUnits = { LightTank2, LightTank3 }
+
+KillCounter = 0
+KillCounterHuntThreshold = 20
+
+WorldLoaded = function()
+	player = Player.GetPlayer("GDI")
+	enemy = Player.GetPlayer("Nod")
+	civilian = Player.GetPlayer("Neutral")
+
+	Trigger.OnObjectiveAdded(player, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "New " .. string.lower(p.GetObjectiveType(id)) .. " objective")
+	end)
+	Trigger.OnObjectiveCompleted(player, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective completed")
+	end)
+	Trigger.OnObjectiveFailed(player, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective failed")
+	end)
+
+	Trigger.OnPlayerWon(player, function()
+		Media.PlaySpeechNotification(player, "Win")
+	end)
+
+	Trigger.OnPlayerLost(player, function()
+		Media.PlaySpeechNotification(player, "Lose")
+	end)
+
+	destroyObjective = player.AddPrimaryObjective("Destroy the Nod ********.")
+
+	Trigger.OnKilled(Airfield, function()
+		player.MarkCompletedObjective(destroyObjective)
+	end)
+
+	Utils.Do(NodBase, function(structure)
+		Trigger.OnKilled(structure, function()
+			player.MarkCompletedObjective(destroyObjective)
+		end)
+	end)
+
+	Trigger.OnAllKilled(IslandSamSites, function()
+		TransportFlare = Actor.Create('flare', true, { Owner = player, Location = Flare.Location })
+		Reinforcements.ReinforceWithTransport(player, 'tran', nil, { lstStart.Location, TransportRally.Location }, nil, nil, nil)
+	end)
+
+	Trigger.OnKilled(CivFleeTrigger, function()
+		Civilian.Patrol(CivFleeRoute, false)
+	end)
+
+	Trigger.OnEnteredFootprint({ CivFleeRoute3.Location }, function(a, id)
+		if a.Owner == civilian then
+			HuntTriggerFunction()
```

Comment:
> maybe make the cat&mouse game more fun: instead of Hunt(), send them (AttackMove) to the location where the commando was when he killed the unit? That way Commando & Chinook could lure forces away from the base

Seems like a good idea.